test/workflows/components/workflows-v1alpha2.libsonnet

```diff
@@ -311,7 +311,7 @@
             $.parts(namespace, name, overrides).e2e(prow_env, bucket).buildTemplate("python-tests", pythonImage, [
               "test/scripts/v1alpha2/python-tests.sh",
             ]),  // run python tests
-            $.parts(namespace, name, overrides).e2e(prow_env, bucket).buildTemplate("run-tests", helmImage, [
+            $.parts(namespace, name, overrides).e2e(prow_env, bucket).buildTemplate("run-tests", "liuhougangxa/golang-gcloud:1.12.8-257.0.0", [
```

Comment:
Why can't we use standard go image here?

Reply:
helmImage is an image with gcloud and golang:1.9, but now after some dependency in vendor upgraded, golang 1.9 cannot work. we need a new image with gcloud and golang 1.10 at least

Reply:
How about using  kubeflow test worker image like in other repos - gcr.io/kubeflow-ci/test-worker:v20190421-fba47fe-e3b0c4 ?

Reply:
I will try if there are such  test worker images in gcr.io/kubeflow-ci can work

Reply:
gcr.io/kubeflow-ci/test-worker can work indeed. Thanks for your comment